File: CodeWars-Solutions/Bit-Counting.py
# Python
def count_bits(n):
    nBin = bin(n)
    nBin = str(nBin)
    count = 0;
    for x in range(0, len(nBin)):
        if (nBin[x:x+1] == "1"):
            count = count + 1
    return count

File: CodeWars-Solutions/Bit-Counting.java
//Java
public class BitCounting {

  public static int countBits(int n){
    // Show me the code!
    String nBin = Integer.toBinaryString(n);
    int count = 0;
    for (int i = 0; i < nBin.length(); i++){
      if (nBin.charAt(i) == '1'){
        count++;
      }
    }
    return count;
  }
  
}
